### PR TITLE
Create ccpp.yml space Ton Arx

### PR DESCRIPTION
**Pull request by Psyruss77**

### CODE REVIEW

**Review by Psyruss — COMMENTED** [review]
build:

      runs-on: ubuntu-latest

      steps:
     - uses: actions/checkout@v1
     - name: configure
       run: ./configure

**Comment by Psyruss** [comment]
build:

      runs-on: ubuntu-latest

      steps:
     - uses: actions/checkout@v1
     - name: configure
       run: ./configure

**Comment by Psyruss** [comment]
access

**Review by Psyruss — COMMENTED** [review]
yes

**Review by Psyruss — COMMENTED** [review]
Let’s Crypt

**Review by Psyruss — COMMENTED** [review]
Start SpaceSunNight

**Review by Psyruss — COMMENTED** [review]
Start SpaceSunNight

File: .github/workflows/ccpp.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: C/C++ CI
```
Comment (Psyruss77) [inline .github/workflows/ccpp.yml:1]: C-Starting

**Review by Psyruss77 — COMMENTED** [review]
Start sudo get #//c:/hd storage /secure app

File: .github/workflows/ccpp.yml (hunk 1 above)
Comment (Psyruss77) [reply .github/workflows/ccpp.yml:1]: > C-Starting 

Ss

**Review by Psyruss77 — COMMENTED** [review]
Start

**Comment by EmelyanenkoK** [comment]
Closed in favor of .github/workflows

**Pull request closed without merging**